managing_providers/_topics/embedded_workflows.md

```diff
@@ -280,6 +280,21 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
   }
 }
 ```
+#### Running an Embedded Workflow on the Openshift Container Platform (OCP) 
+
+If the user is running an embedded workflow on OCP, as opposed to appliances, and is using a docker repository requiring a custom pull secret for their container images, the following steps are necessary:
+
+1. create a docker pull secret for the repository in question in the same namespace as IM, for example: 
```

Comment:
"IM" is a downstream specific name, so we should use a conref here.  Also, minor, but this should probably be a full sentence.

```suggestion
1. Create a docker pull secret for the repository in question in the same namespace as {{ site.data.product.title_short }}, for example:
```

Reply:
It says "docker pull secret", but that doesn't sound right to me - I think it should just be "Create a pull secret".  Maybe I'm wrong though - @agrare Thoughts?

Reply:
Technically you are creating a k8s `Secret` object and the purpose is up to the user, but I think `Pull Secret` or `Image Pull Secret` would be an accurate description.